ui/lib/core/addon/components/stat-text.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
I think this is the right move, adding it to addons because we do want to use it in replication, which is in another engine. But normally, you wouldn't do this unless you knew it needed to be consumed in another engine within the app. e.g. KMIP, replication, or open-api-explorer (which all live outside of the app directory).